packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -23,21 +23,123 @@ Okta Identity Engine currently supports Authenticators for the following factors
 * Phone (SMS, Voice Call)
 * Email
 * WebAuthn
+* Duo
+* Custom App
 
 ## Get started
 
-Explore the Authenticators Administration API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/e357f5e0a1cf3be4c20e)
+Explore the Authenticators Administration API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/836eb57018cba45da121)
 
 ## Authenticators Administration operations
 
 The Authenticators Administration API has the following CRUD operations:
 
+* [Create Authenticator](#create-authenticator)
 * [List Authenticators](#list-authenticators)
 * [Get an Authenticator by ID](#get-an-authenticator-by-id)
 * [Update Authenticator settings](#update-authenticator-settings)
 * [Activate an Authenticator](#activate-an-authenticator)
 * [Deactivate an Authenticator](#deactivate-an-authenticator)
 
+### Create Authenticator
+
+<ApiOperation method="post" url="/api/v1/authenticators" />
+
+Create an Authenticator
+
+#### Request path parameters
+
+N/A
+
+#### Request query parameters
+
+| Parameter | Description                                                                                           | Param Type | DataType      | Required | Default |
+| --------- | ----------------------------------------------------------------------------------------------------- | ---------- | ------------- | -------- | ------- |
+| activate  | Executes [activation lifecycle](#activate-an-authenticator) operation when creating the authenticator | Query      | Boolean       | FALSE    | TRUE    |
+
+#### Request body
+
+An [Authenticator Object](#authenticator-object) that needs to be created
+
+#### Response body
+
+The created [Authenticator](#authenticator-object)
+
+#### Use example
+
+Returns the created Authenticator with an `id` value of `aut9gnvcjUHIWb37J0g4`:
+
+#### Request
+
+```json
+{
+    "key": "duo",
+    "name": "Duo Security",
+    "provider": {
+        "type": "DUO",
+        "configuration": {
+            "userNameTemplate": {
+                "template": "oktaId"
+            },
+            "integrationKey": "testIntegrationKey",
+            "secretKey": "testSecretKey",
+            "host": "{Duo Org Hostname}"
```

Comment:
We should call this DUO Api hostname to match what you see in DUO's website when you configure the Okta app

<img width="881" alt="image" src="https://user-images.githubusercontent.com/33136962/179637899-6ad667bb-10b8-45a1-8d1a-93838ea9e2dc.png">

Reply:
I thought about that, but we call it `The Duo Security org hostname` [here in the dev docs](https://developer.okta.com/docs/reference/api/authenticators-admin/#authenticator-properties). So I changed it to be that. If needed, I can change both to API Hostname. We call it API Hostname in the product help.